config.json

```diff
@@ -6,63 +6,63 @@
       "name": "radio-driver",
       "driver_class": "RadioDriver",
       "device_id": "RADIO",
-      "enabled": "true"
+      "enabled": true
     },
 
     {
       "name": "ping-driver",
       "driver_class": "PingDriver",
       "device_id": "MISC_RADIO_1",
-      "enabled": "true"
+      "enabled": false
     },
 
     {
       "name": "camera-driver",
       "driver_class": "CameraDriver",
       "device_id": "CAMERA_1",
-      "enabled": "true"
+      "enabled": true
     },
 
     {
       "name": "cutdown-driver",
       "driver_class": "CutdownDriver",
       "device_id": "CUTDOWN",
-      "enabled": "true"
+      "enabled": true
     },
 
     {
       "name": "gps-driver",
       "driver_class": "GPSDriver",
       "device_id": "GPS",
-      "enabled": "true"
+      "enabled": true
     },
 
     {
       "name": "science-driver",
       "driver_class": "ScienceDriver",
       "device_id": "MISC_SENSOR_1",
-      "enabled": "true"
+      "enabled": true
```

Comment:
Should `"enabled"` for the science driver be set to `false`, since you removed the `self.data_transmit(row)` line in the science driver for the time being?

Reply:
ehhhh, i mean usually we only really change the config file when we decide what drivers we need for launch.  The driver technically still works, it can log data to disk.  Transmit will come later.  Guess it doesn't super matter.  But also I'd like to hear what Aryan thinks

Reply:
I think it doesn't matter too much either way. If we still want it to log to disk at least, then we can keep it as true